src/test/java/com/amazon/ion/TextToBinaryUtils.kt

```diff
@@ -64,4 +81,29 @@ object TextToBinaryUtils {
     fun ByteArray.byteArrayToBitString(): String {
         return this.joinToString(" ") { it.toUByte().toString(2).padStart(8, '0') }
     }
+
+    /**
+     * Converts a byte array to a string of hex bytes, such as "A5 0F EC 52".
+     * The purpose of this method is to make it easier to read and write test assertions.
+     */
+    @JvmStatic
+    fun ByteArray.byteArrayToHexString(): String {
+        return this.joinToString(" ") { it.toUByte().toString(16).padStart(2, '0') }
+    }
```

Comment:
Just use `ByteArray.toHexString()`, provided in the Kotlin standard library, instead of adding this method.